introduction.md

```diff
@@ -1,3 +1,9 @@
 Hi, I'm Nirina, glad to restart the challenges for mastering Github !
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?